pages/common/mocha.md

```diff
@@ -0,0 +1,15 @@
+# mocha
+
+> Execute Mocha JavaScript test runner
+
+- Run tests with default configuration, or as configured in `mocha.opts`
+
+`mocha`
+
+- Run tests that match a specific grep pattern
+
+`mocha --grep {{^regex$}}`
+
+- Run tests every time files change until you kill it
```

Comment:
According to the mocha docs it's: "Executes tests on changes to JavaScript in the CWD, and once initially."

So how about something like: "Run tests on changes to JavaScript files in the current directory and once initially"?

I'll commit that for now and if you'd like it changed just let me know.

Thanks.